maps/away/ships/unathi_pirate/hiskyn/unathi_pirate_hiskyn_ghostroles.dm

```diff
@@ -0,0 +1,54 @@
+/datum/ghostspawner/human/hiskyn
+	short_name = "hiskyn"
+	name = "Hiskyn's Revanchist Crewman"
+	desc = "Serve aboard a ship of the pirate fleet Hiskyn's Revanchists - former Kazhkz-Han'san Unathi who refused to join the Empire of Dominia. Obey your captain, defend your ship and crew, and keep an eye out for opportunities to shed some Kazhkz-Han'san blood! NOT AN ANTAGONIST! Do not act as such."
```

Comment:
all ghost spawners should have separate desc and welcome_message
see goon base for an example